docs/UserGuide.md

```diff
@@ -3,176 +3,203 @@ layout: page
 title: User Guide
 ---
 
-AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
+# Addendum for Teaching Assistants (ATAS) - User Guide
 
-* Table of Contents
-{:toc}
+ATAS is a **desktop app for managing students’ particulars, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type really quickly, ATAS will help you to manage your contacts and students’ particulars faster than traditional GUI apps.
+<hr>
+--------------------------------------------------------------------------------------------------------------------
 
+## Table of Contents
+
+* [Quick start](#qs)
+* [Features](#features)
+  * [Viewing help](#help) (coming soon)
+  * [Adding a students](#add)
+  * [Listing all students](#list)
+  * [Locating students by name](#find)
+  * [Deleting a student](#delete)
+  * [Clearing all entries](#clear)
+  * [Exiting the program](#exit)
+  * [Saving data](#sd)
+  * [User confirmation prompt](#ucp) (coming soon)
+* [FAQ](#faq)
+* [Command summary](#cs)
 --------------------------------------------------------------------------------------------------------------------
 
-## Quick start
+## <a name="qs"></a>Quick start
 
 1. Ensure you have Java `11` or above installed in your Computer.
 
-1. Download the latest `addressbook.jar` from [here](https://github.com/se-edu/addressbook-level3/releases).
+2. Download the latest `atas.jar` from [here](https://github.com/AY2021S1-CS2103T-W16-4/tp/releases). 
 
-1. Copy the file to the folder you want to use as the _home folder_ for your AddressBook.
+3. Copy the file to the folder you want to use as the _home folder_ for your ATAS.
 
-1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
-   ![Ui](images/Ui.png)
+4. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
+   [insert image of GUI here]
 
-1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
+5. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
    Some example commands you can try:
 
-   * **`list`** : Lists all contacts.
+   * **`list`** : Lists out all the contacts.
```

Comment:
Perhaps "contacts" can be changed to "students"?

Reply:
Thanks for pointing this out, as well as in other places!